packages/react/src/themes/teams/components/Attachment/attachmentStyles.ts

```diff
@@ -68,19 +67,12 @@ const attachmentStyles: ComponentSlotStylesPrepared<AttachmentProps, AttachmentV
     })
 
     return {
-      [`& .${Icon.className}`]: {
-        color: v.textColor, // this breaks the color change on hover
```

Comment:
I can't see why this was needed. Would love to know if I'm missing something. Here is the original PR where it was added... https://github.com/microsoft/fluent-ui-react/commit/eb47a325c0a410060ec99a66ae3702c4ba426107